fenix/app/onboarding.fml.yaml

```diff
@@ -32,16 +32,16 @@ features:
 
             notification-permission:
               card-type: notification-permission
-              title: juno_onboarding_enable_notifications_title
-              body: juno_onboarding_enable_notifications_description
+              title: onboarding_home_enable_notifications_title_nimbus
+              body: onboarding_home_enable_notifications_description_nimbus
               ordering: 30
               primary-button-label: juno_onboarding_enable_notifications_positive_button
               secondary-button-label: juno_onboarding_enable_notifications_negative_button
 
     defaults:
       - channel: developer
         value:
-          enabled: false
+          enabled: true
```

Comment:
umm for other features this would make sense but since it's just us working on this, it might interfere with the dev flow for other folks.

Reply:
Yep, fair point 👍